utils/generateTokensArray.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+
+const tokenGroupsDir = path.join(__dirname, '../src/tokens/token-groups');
+
+const generateTokensArray = () => {
+  /**
+   * Allowed Polaris token custom properties.
+   *
+   * Result: ['--p-background', '--p-text', etc...]
+   */
```

Comment:
**suggestion:** Should we move this code block above the `generateTokensArray` util?